tiledb/tests/test_enumeration.py

```diff
@@ -93,7 +93,7 @@ def test_array_schema_enumeration(self):
             with self.assertRaises(tiledb.TileDBError) as excinfo:
                 assert A.enum("enmr3") == []
             assert (
-                "ArraySchema: Unable to check if unknown enumeration is loaded. No enumeration named 'enmr3'."
+                "Array: Unable to get enumeration; Enumeration 'enmr3' does not exist."
```

Comment:
Until 2.27 releases we could use a different expected error message depending on `libtiledb.version()`, and switch to only the new one afterwards.

Reply:
You are right Teo. Thanks for the suggestion!